src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -18,8 +22,61 @@
  */
 public class SampleDataUtil {
```

Comment:
Shouldn't this be in test?

Reply:
Same as above. The data is for initializing the main app in non-testing mode.

Reply:
Done! Changed editor code reformat setting -  for those curious: Editor/Code Style/Java/Tabs and Indents/Continuation Indent = 8.